udata_front/views/organization.py

```diff
@@ -86,6 +86,9 @@ def get_context(self):
         if self.organization.deleted and not can_view.can():
             abort(410)
 
+        dataservices = Dataservice.objects(
```

Comment:
I think you need to filter these out if not visible.
Since we're using the search service, I guess we won't be able to display these if the user is allowed.